lib/watch.js

```diff
@@ -59,6 +59,20 @@ const pushFile = (gateway, syncedFilePath) => {
   });
 };
 
+const deleteFile = (gateway, syncedFilePath) => {
+  let filePath = filePathUnixified(syncedFilePath); // need path with / separators
+  const formData = {
+    path: filePath,
+    primary_key: filePath,
+  };
+
+  return gateway.delete(formData).then(body => {
+    if (body) {
+      logger.Success(`[Sync] Deleted: ${filePath}`);
```

Comment:
Hmmm, maybe it should be `Info` or smth to be more visible on the usual green wall of success text ;)